src/logic/operate.js

```diff
@@ -0,0 +1,30 @@
+import Big from 'big.js';
+
```

Comment:
- [ ] Kindly add a test for this module, since this is mainly the one to being tested

Reply:
@BregornOriginal thank you for your review...
I was the only one available for the week coding, I do not have a coding partner. I sent a message to student success, I have not received a reply yet, and I would need help working on the other test.